src/main/java/org/sonarqube/gradle/AndroidUtils.java

```diff
@@ -74,11 +78,33 @@ static void configureForAndroid(Project project, String userConfiguredBuildVaria
     BaseVariant variant = findVariant(project, userConfiguredBuildVariantName);
     if (variant != null) {
       configureForAndroid(project, variant, properties);
+      configureAndroidLintReport(project, variant, properties);
```

Comment:
Wouldn't it be better placed in `configureForAndroid` since it's part of the android configuration?

Reply:
yes make sense, especially with the addition of configureTestReports